usb/usb.go

```diff
@@ -253,7 +255,7 @@ func buildKernel() error {
 		fmt.Printf("copying %v to linux-stable/.config: %v", *config, err)
 	}
 
-	cmd := exec.Command("make", "--directory", "linux-stable", "-j64")
+	cmd := exec.Command("make", "--directory", "linux-stable", "-j" + strconv.Itoa(runtime.NumCPU()))
```

Comment:
I've seen recommendations to use more jobs than CPUs on the basis that some jobs may block on disk. Something like `n+1` or `2n` would work; and, generally having too many jobs is not nearly as bad on runtime as having too few.

Reply:
Yeah it's hard to get the profiling right. I think 2n is a bit aggressive, and just n leaves some room for web browsing at the same time.

Let me add a constant to set additional threads and we can tune it as necessary.